sphinx_doc_src/cmds/source.rst

```diff
@@ -17,6 +17,8 @@ Description
 
 ``source`` evaluates the commands of the specified file in the current shell. This is different from starting a new process to perform the commands (i.e. ``fish < FILENAME``) since the commands will be evaluated by the current shell, which means that changes in shell variables will affect the current shell. If additional arguments are specified after the file name, they will be inserted into the ``$argv`` variable. The ``$argv`` variable will not include the name of the sourced file.
 
+fish will search the working directory to resolve relative paths but unlike bash will not search ``$PATH``.
```

Comment:
We typically use "unlike other shells". The docs even mention it as a "magic phrase" that you can search for at one point.

Reply:
"unlike other shells" makes it sounds like the behavior is unique to `fish`, but it is also `like` some other shells in this regard. `ksh` and `tcsh` don't search `$PATH`, but `bash` and `zsh` do.  I don't think we need to go into that level of detail here, but thought it was worth mentioning that there was a difference with the most popular shell shere.

Reply:
Yeah I think "unlike other shells" here makes this seem like some departure, when we're just like other non-Bourne shells that don't do it along with csh and ksh. Honestly I'd just make this whole thing simpler and not mention the bit about other shells being different. Just mention that PATH is not searched.

Reply:
So shells that check $PATH, from a quick `source ls` (and corresponding error):

- bash
- zsh
- mksh
- dash (with `.`, it doesn't have source)

With that alone, you have the overwhelming majority of our users covered. And even those that come from other shells that might not do this would know that we might be talking about all those others.